index.html

```diff
@@ -33,140 +40,210 @@ <h1>I’m Ismail Glad to see you!</h1>
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javaScript</a></li>
                     </ul>
+                    <button>See Project</button>
                 </div>
-                <div class="categories">
-                    <ul>
-                        <li>
-                            <p>Frameworks</p>
-                            <i class="fa fa-angle-right"></i>
-                        </li>
+            </section>
+
+            <section class="row work">
+                <div class="col-ss-7 mob1">
+                    <img src="images/Snapshoot-Portfolio-4.png" class="mob1" alt="">
+                </div>
+                <div class="col-ss-5">
+                    <h2>Uber Navigation</h2>
+                    <div class="counter mob1">
+                        <span>CANOPY</span>
+                        <span class="counters"></span>
+                        <span>Back End Dev</span>
+                        <span class="counters"></span>
+                        <span>2015</span>
+                    </div>
+                    <div class="counter desk1">
+                        <span>Uber</span>
+                        <span class="counters"></span>
+                        <span>Lead Developer</span>
+                        <span class="counters"></span>
+                        <span>2018</span>
+                    </div>
+                    <p class="mob1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <p class="desk1">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                    <ul class="mob1">
+                        <li><a href="#">HTML</a></li>
+                        <li><a href="#">CSS</a></li>
+                        <li><a href="#">JavaScript</a></li>
                     </ul>
-                    <hr>
-                    <ul>
-                        <li>
-                            <p>Skills</p>
-                            <i class="fa fa-angle-right"></i>
-                        </li>
+                    <ul class="desk1">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">Ruby on rails</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javaScript</a></li>
                     </ul>
-                    <hr>
+                    <button>See Project</button>
+                </div>
+                <div class="col-ss-7">
+                    <img src="images/Snapshoot-Portfolio-2.png" class="desk1" alt="">
+                </div>
+            </section>
+
+            <section class="row about-myself">
+                <div class="col-ss-5">
+                    <p class="myself">About Myself</p>
```

Comment:
- [ ] For semantic reasons, kindly make sure you use a heading element here `h2...x` instead of a generic paragraph

Reply:
Fixed.